pkg/container/qemu_runner.go

```diff
@@ -402,7 +402,7 @@ func (bw *qemu) WorkspaceTar(ctx context.Context, cfg *Config) (io.ReadCloser, e
 		nil,
 		outFile,
 		false,
-		[]string{"sh", "-c", "cd /home/build && tar cvpzf - --xattrs --acls *"},
+		[]string{"sh", "-c", "cd /home/build && tar cvpzf - --xattrs --acls --exclude='*fifo*' *"},
```

Comment:
Should we include `dpkg` in the list of wolfi packages we build on PRs both to confirm this works, but also as a regression test?

Reply:
right, will add :+1:

Reply:
done